asapdiscovery-alchemy/asapdiscovery/alchemy/tests/test_cli.py

```diff
@@ -245,3 +246,148 @@ def test_alchemy_prep_run_all_pass(tmpdir, mac1_complex):
         assert len(prep_dataset.input_ligands) == 5
         assert len(prep_dataset.posed_ligands) == 5
         assert prep_dataset.failed_ligands is None
+
+
+def test_alchemy_predict_no_experimental_data(tyk2_result_network, tmpdir):
+    """Test predicting the absolute and relative free energies with no experimental data, interactive reports should
+    not be generated in this mode.
+    """
+
+    runner = CliRunner()
+
+    with tmpdir.as_cwd():
+        # write the results file to local
+        tyk2_result_network.to_file("result_network.json")
+
+        result = runner.invoke(
+            alchemy,
+            [
+                "predict",
+            ],
+        )
+        assert result.exit_code == 0
+        assert "Loaded FreeEnergyCalculationNetwork from" in result.stdout
+        assert "Absolute predictions written" in result.stdout
+        assert "Relative predictions written" in result.stdout
```

Comment:
the full sentence (i.e. with the filename after 'written') was failing in test because the `result.stdout` has varying newline characters that would break the `in` statement.. truncated like this should be fine I think!